quilt/test/test_install.py

```diff
@@ -125,3 +125,15 @@ def _mock_package(self, package, pkg_hash, contents, hashes):
     def _mock_s3(self, pkg_hash, contents):
         s3_url = 'https://example.com/%s' % pkg_hash
         self.requests_mock.add(responses.GET, s3_url, contents)
+
+# execute outside of mock scope so we get real network
```

Comment:
Uh, unit tests should never hit real servers. You can add "integration test", but it should be a separate file - not something that runs as a part of `pytest`.